src/analysis/raster/qgsrastercalcnode.cpp

```diff
@@ -360,6 +382,15 @@ QString QgsRasterCalcNode::toString( bool cStyle ) const
       break;
     case tMatrix:
       break;
+    case tFunction:
```

Comment:
@Franc-Brs you need to add the `if ( cStyle )` branch too or this won't work with acceleration on.

The `cStyle` implementation must return a valid `C++` source for the condition that will be compiled from OpenCL implementations.

Please make sure you test this with OpenCL enabled, the current status of this (already merged :cry:  ) PR is that it introduces a regression.

Reply:
thanks @elpaso and no worries, it is not merged yet ;-)

@Franc-Brs could you please update the PR so that it includes a version for OpenCL, i.e. when `cStyle == true` - I think it should be possible to use the ternary operator (`a ? b : c`) like in C/C++ according to the docs:
https://www.khronos.org/registry/OpenCL/specs/2.2/html/OpenCL_C.html#operators-ternary-selection

Reply:
> thanks @elpaso and no worries, it is not merged yet ;-)
> 
> @Franc-Brs could you please update the PR so that it includes a version for OpenCL, i.e. when `cStyle == true` - I think it should be possible to use the ternary operator (`a ? b : c`) like in C/C++ according to the docs:
> https://www.khronos.org/registry/OpenCL/specs/2.2/html/OpenCL_C.html#operators-ternary-selection

Hi, thanks @elpaso  and thanks @wonder-sk I'll do it asap :)